components/molecule/tabs/src/components/MoleculeTabs.js

```diff
@@ -42,20 +42,28 @@ const MoleculeTabs = ({
 
   const activeTabContent = childrenArray.reduce((activeContent, child) => {
     if (child) {
-      const {children: childrenChild, active} = child.props
-      return active ? childrenChild : activeContent
+      const {children: childrenChild, active, numTab} = child.props
+      return active ? (
+        <div
+          className={CLASS_CONTENT}
+          id={`molecule-tab-content-${numTab}`}
```

Comment:
If we add two tabs in the same page. Is this id going to be repeated ?

Reply:
For sure, yes. Good catch. 
I've added this to help the `aria-controls` on tab elements 🤔. 

In order to do it unique, we must pass a `id` to the MoleculeTab and use it to build it dynamically, isn't it? What do you think?

Reply:
Pass an id to `MoleculeTabs`, prop drill it to the MoleculeTab and concat the numtab to get the id of each.